### ISSUE TITLE
JSON files uploader input element looks little constricted in its appearance.

### ISSUE THREAD

**Issue by Princu7** [fossasia/open-event-webapp#759]
The default input element which pops up after selecting the Upload JSON files radio button has somewhat constricted appearance. Made a js fiddle for it. Here is how it looks after making small change
https://jsfiddle.net/7q2efwdz/

**Comment by mariobehling** [comment]
Any screenshot what you mean?

**Comment by Princu7** [comment]
This is how the original JSON files upload element look like
![screenshot from 2016-08-31 21-43-08](https://cloud.githubusercontent.com/assets/8847265/18136937/24310cd2-6fc4-11e6-9351-7979087fde36.png)
And after dong a small change it looks like the above js fiddle link.

**Comment by aayusharora** [comment]
Current snapshot in Chrome
![09](https://cloud.githubusercontent.com/assets/12194719/18137072/9dd1cd92-6fc4-11e6-9ac0-b1fe072770c4.png)
Current Snapshot of Fiddle  in chrome
According to me, it is a browser issue in Firefox.

@Princu7 Please check your fiddle on chrome, it look like it shift above in Chrome while it was right earlier.

**Comment by Princu7** [comment]
Sorry. Don't have chromium installed as of currently. Currently downloading it but it will take some time. But most probably it is the problem with the browser itself.

**Comment by championswimmer** [comment]
We can still increase the height a little for accomodate for both browsers

**Comment by Princu7** [comment]
That's a good idea. I will work on the issue.

**Comment by Princu7** [comment]
Sorry for the little delay. So I searched on the net regarding this problem but failed to find something substantial. So I played around a little more to get the input element to behave the same across the two browsers.I ended up changing some properties and now the button looks the same across both the browsers.
Here is the js fiddle.
https://jsfiddle.net/7q2efwdz/1/
And here are the screenshots  of both the browsers.
In chrome
![screenshot from 2016-09-02 16-33-09](https://cloud.githubusercontent.com/assets/8847265/18202081/8b5cd5ea-712b-11e6-8641-c219cfbc6ddb.png)
In firefox
![screenshot from 2016-09-02 16-36-18](https://cloud.githubusercontent.com/assets/8847265/18202089/94ff5b40-712b-11e6-9ac9-220233f755fc.png)
I will send a pull request. Tell me if this causes any side problems.
